dcos/dcos-ui
# DCOS-57501: [UI] show confirmation dialog before installing community package

Add a confirmation before running
community packages.

Closes https://jira.mesosphere.com/browse/DCOS-57501

## Testing
1. Go to Catalog tab.
2. Scroll down and find a community package.
3. Click it. The package detail page should be visible.
4. Click "Review & Run".
5. Verify that a confirmation modal is shown.
6. Verify that clicking cancel or closing the modal brings us back to the package detail page.
7. Verify that clicking continue brings us to the package configuration page.
8. Verify that this isn't shown for certified packages.
9. Verify that the added tests make sense.

## Trade-offs
See my comments.

## Dependencies
None.

## Screenshots
![Screenshot from 2019-08-21 13-37-08](https://user-images.githubusercontent.com/40791275/63433261-837a9f00-c42b-11e9-99c8-74c2858d7ba6.png)
